import-scripts/backup-eks-dbs.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+DBS=("cgds_gdac" "cgds_triage" "keycloak")
+LOCAL_BACKUP_DIR=/data/mysql-dumps
+SLACK_URL=`cat /data/portal-cron/pipelines-credentials/mskcc-sysadmin-slack.url`
+
+for db in ${DBS[@]}; do
+  DUMP_FAILURE=0
+  echo "Backing up: '${db}'"
+
+  PORTAL_INFO_TABLE=$(mysql --login-path=mysql_localhost -e "SHOW TABLES LIKE 'info';" $db | tail -n1)
+  if [ ! -z "$PORTAL_INFO_TABLE" ]; then
+    DB_SCHEMA_VERSION=".v$(mysql --login-path=mysql_localhost -e "SELECT db_schema_version FROM info;" $db | tail -n1)"
+  else
+    DB_SCHEMA_VERSION=""
+  fi
+  SQLDUMP_FILENAME=${db}.$(date +%Y%m%d)${DB_SCHEMA_VERSION}.sql.gz
+  SQLDUMP_FULLPATH=${LOCAL_BACKUP_DIR}/$SQLDUMP_FILENAME
+
+  # The return status of a pipeline is the exit status of the last command, unless the pipefail option is enabled. If pipefail is enabled, the pipeline's return status is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands exit successfully.
+  $(set -o pipefail && mysqldump --login-path=mysql_localhost --quick $db | gzip > $SQLDUMP_FULLPATH)
+  if [ $? -eq 0 ]; then
```

Comment:
I guess in the case of a blank/empty database, this is where we would notice a failure (although an empty database maybe would be "successfully" dumped here.